### ISSUE TITLE
[Request]: Aleph One (Marathon Infinity)

### ISSUE THREAD

**Issue by GriekseEi** [Calinou/scoop-games#990]
### Prerequisites

- [X] I have searched all issues/PRs to ensure it has not already been reported or fixed.

### Criteria

- [X] Reasonably well-known and widely used
- [X] A game or a game-related tool
- [X] English interface (or at least English documentation)
- [X] Latest stable or development version
- [X] Full version (i.e. not a trial version)
- [X] Fairly standard install (e.g. uses a version-specific download URL, no elaborate pre/post install scripts)

### Name

Aleph One (Marathon Infinity)

### Description

The freeware first-person shooter Marathon Infinity by Bungie Software and Double Aught, packaged in the Aleph One sourceport.

### Homepage

https://alephone.lhowon.org/

### Download Link(s)

https://github.com/Aleph-One-Marathon/alephone/releases/download/release-20230529/MarathonInfinity-20230529-Win.zip

### Some Indication of Popularity/Repute

520 stars on GitHub

**Comment by Calinou** [comment]
Should we have one manifest for all Marathon games, or one manifest per game? The game data is quite small by today's standards, so it might make sense to bundle them together.

**Comment by GriekseEi** [comment]
> Should we have one manifest for all Marathon games (1/2/Infinity), or one manifest per game? The game data is quite small by today's standards, so it might make sense to bundle them together.

Yeah, you are right. All three games end up being updated at the same time anyways. I'll close these issues and the accompanying PR, and will make a new one for just the whole trilogy itself.
